recipes/pep8/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.7.0" %}
+
+package:
+  name: pep8
```

Comment:
@pmlandwehr do you want to get this merged, and then add `pycodestyle` as @jjhelmus suggested or do you want to change it here and now?